src/urls/main.lua

```diff
@@ -76,7 +76,7 @@ do
   
   function k.open(url, mode)
     checkArg(1, url, "string")
-    checkArg(2, mode, "number")
+    checkArg(2, mode, "string")
```

Comment:
This broke TTY's `printk` function: `/boot/kernel.lua:315: bad argument #2 to 'open' (string expected, got number)`

No clue how to fix this myself

Reply:
whoops, will fix